src/fitnesseMain/FitNesseMain.java

```diff
@@ -49,6 +49,30 @@ public static void main(String[] args) {
   protected static void exit(int exitCode) {
     System.exit(exitCode);
   }
+  
+  // Testing code anaylsis tools
+  public String testMethod(){
+    
+  }
+  
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '{' at column 5 should be on the previous line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Aalone)